uo_sys_framework/ACREModule/cfgFunctions/common/fn_addRack.sqf

```diff
@@ -0,0 +1,148 @@
+/*	Description: Adds racks to objects as specified.
+ *	Arguments:
+ * 0: vehicle or object <OBJECT> (default: objNull)
+ *	Return Value:
+ * 		NULL
+ *	Author
+ *		TrainDoctor
+*/
+
+#include "\x\UO_FW\addons\main\script_macros.hpp"
+
+if (!UO_FW_ACRE_SETTINGS_ENABLED) exitwith {};
+	
+if (!isServer) exitWith {
+	(format ["(ACRE_SETUP_MODULE) [ERR]: NO FALLBACK ERROR ENCOUNTERED"]) call UO_FW_FNC_DebugMessage;
+	(format ["(ACRE_SETUP_MODULE) [ERR]: THIS SCRIPT MUST BE RUN SERVER-SIDE ONLY"]) call UO_FW_FNC_DebugMessage;
```

Comment:
Think I am going to redefine the CBA error/logging macros with an ability to display on the UO_FW error display so an ERROR_1() macro will also call `UO_FW_FNC_DebugMessage` if FW debugmode is enabled

Reply:
ERROR_1() should remain the same. Ensure that UO_FW_FNC_DebugMessage also performs the same function as ERROR_1(). ACE/CBA compliance.